src/lib.rs

```diff
@@ -110,8 +96,9 @@ fn internal_generate(mut args: GenerateArgs) -> Result<PathBuf> {
     let mut config = Config::from_path(
         &locate_template_file(CONFIG_FILE_NAME, &template_base_dir, &template_dir).ok(),
     )?;
-    config.template.get_or_insert(Default::default());
 
+    // the `--init` parameter may also be set by the template itself
+    // TODO: should we show a warning to the user if the template doesn't agree with the parameter? The user might not expect the template files to end up in CWD!
```

Comment:
@taurr shall we create a followup tracking issue for that `TODO` maybe?

Reply:
#763